lib/cisco_node_utils/yum.rb

```diff
@@ -20,8 +20,15 @@
 require_relative 'node_util'
 
 module Cisco
-  # This Yum class provides cisco package management functions through nxapi.
+  # This Yum class provides cisco package management functions.
   class Yum < NodeUtil
+    EXEC_IN_DEFAULT_NS = 'ip netns exec default'
+
+    # This function accepts name of the rpm package and returns a match group.
+    # Match group when platform is ios_xr :
+    #   1. package name, 2. package version, 3. os version, 4. platform
+    # Match group when platform is nexus :
+    #   1. package name 2. package version 3. platform
```

Comment:
My point was that you said that platform nexus has three match groups, but in the name_arch_regex it only has two - so this comment is incomplete/inaccurate.